compiler/src/dmd/root/string.d

```diff
@@ -20,24 +20,13 @@ inout(char)[] toDString (inout(char)* s) pure nothrow @nogc
     return s ? s[0 .. strlen(s)] : null;
 }
 
-private struct FTuple(T...)
-{
-    T expand;
```

Comment:
does `alias expand this;` work here? if so we should use it and give `FTuple` a less cryptic name.

Reply:
I tried, but they won't auto expand in C-style variadic argument lists

Reply:
Perhaps `str.lengthPtr.expand` is clearer?